charts/foreman/README.md

```diff
@@ -0,0 +1,83 @@
+|--------------------------|---------------------------------|-----------------------|
+| `ingress.className`      | Memory limits for the container | `nginx`               |
+| `ingress.host`           | Memory limits for the container | `foreman.dev.local`   |
+| `ingress.path`           | Memory limits for the container | `/`                   |
+| `ingress.annotations`    | Memory limits for the container | `{}`                  |
+| `ingress.tls.secretName` | Memory limits for the container | `foreman-ingress-tls` |
+
+### Config parameters
+
+| Name                            | Description                                                                | Value   |
+|---------------------------------|----------------------------------------------------------------------------|---------|
+| `config.existingSecret`         | Name of an existing secret containing the configuration                    | `""`    |
+| `config.existingConfigMap`      | Name of an existing configmap containing the configuration                 | `""`    |
+| `config.cronJob.namespace`      | Namespace where the CronJob is deployed                                    | `""`    |
+| `config.cronJob.name`           | Name of the CronJob                                                        | `""`    |
+| `config.cookies.key`            | Key which is used to encrypt the session cookie                            | `""`    |
+| `config.cookies.naxAge`         | Max age of the session cookie, e.g. "24h" or "7 days"                      | `""`    |
+| `config.auth.local.enabled`     | Enable local authentication                                                | `false` |
+| `config.auth.local.username`    | Username of the local admin user                                           | `""`    |
+| `config.auth.local.password`    | Password of the local admin user                                           | `""`    |
+| `config.auth.oidc.enabled`      | Enable OIDC authentication                                                 | `false` |
+| `config.auth.oidc.issuer`       | Issuer of the OIDC provider                                                | `""`    |
+| `config.auth.oidc.clientId`     | Client ID of the OIDC provider                                             | `""`    |
+| `config.auth.oidc.clientSecret` | Client Secret of the OIDC provider                                         | `""`    |
+| `config.auth.oidc.publicUrl`    | Public URL of the foreman instance                                         | `""`    |
+| `config.gitlab.host`            | Optional: URL of the GitLab instance (if GitLab features should be active) | `""`    |
+
+### Other parameters
+
+| Name                 | Description                  | Value |
+|----------------------|------------------------------|-------|
+| `replicaCount`       | Number of replicas to deploy | `2`   |
+| `serviceAccountName` | Service Account Name         | `""`  |
```

Comment:
We should explain that this service account is used by Foreman to access the Kubernetes API, and how to configure a service account with access to Renovate.

Reply:
I think that should be part of foreman's wiki. We should add a link when the wiki exists.

Reply:
Alright, makes sense to me.